conda.recipe/meta.yaml

```diff
@@ -19,16 +19,16 @@ requirements:
     - python
     - setuptools
     - cython
-    - numpy x.x
```

Comment:
This is a special syntax that will test against the numpy version specified in https://github.com/Unidata/netcdf4-python/blob/master/appveyor.yml#L15

Because AppVeyor is quite slow we were testing only one, but someone could create a test matrix from it.

Reply:
OK, added that back in.